blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -228,8 +278,6 @@ export class WorkspaceSearch {
       }
```

Comment:
I think this binding of the escape key to close is problematic for if someone doesn't want close on escape key.

Reply:
I suppose we were never supporting not closing with Escape. We could add another property on WS search that we reference in here for whether to close on escape.

Reply:
That seems like a rare use case and not going to support for this version.